app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -61,6 +61,21 @@ def phone
     Phonelib.parse(ial2_data.phone).e164
   end
 
+  def dob
+    return if ial2_data.dob.blank?
+
+    american_date_format = IdentityConfig.store.
+      dob_international_format_opt_out_list.include?(identity.service_provider)
+
+    date = Date.parse(ial2_data.dob)
+
+    if american_date_format
+      date.strftime('%m/%d/%Y')
+    else
+      date.to_s
+    end
+  end
+
   def address
     return nil if ial2_data.address1.blank?
```

Comment:
not part of your pr but how about `return if` here?

Reply:
Do you mean change `return nil if ...` to `return if ...`?